### PR TITLE
Rename webdriver package in composer.json

### LINKED ISSUE #6185: [tools/install.sh] Error in install process on CentOS: "projectname: unbound variable"
**Describe the bug**
On CentOS i'm trying to run the install.sh script in tools and it fails because of the error:
```
[RuntimeException]                                                                     
  Source directory /var/www/Loris/vendor/facebook/webdriver has unpushed changes on the  
   current branch:                                                                       
  M	lib/WebDriverTargetLocator.php    
```

Also worth noting that the `composer update` command gives this output:
```
Package facebook/webdriver is abandoned, you should avoid using it. Use php-webdriver/webdriver instead.
Package zendframework/zend-diactoros is abandoned, you should avoid using it. Use laminas/laminas-diactoros instead.
Package phpunit/phpunit-mock-objects is abandoned, you should avoid using it. No replacement was suggested.

```

### LINKED ISSUE #6220: facebook module version in composer.json
**Describe the bug**
If you do `composer install` then `make`, you will have
image

**To Reproduce**
Steps to reproduce the behavior (attach screenshots if applicable):
1. Do `make dev` or `composer install`
2. Do `make` you will have an error

**What did you expect to happen?**

**Server Environment (if known):**
- LORIS Version: 23

### PR DESCRIPTION
**Pull request by johnsaigle**
## Brief summary of changes

The `facebook/webdriver` package has been changed/renamed to `php-webdriver/webdriver` and as a result our current composer set up is causing errors and warnings in some cases.

This PR corrects the package name. It also uses a tagged release from that repo instead of relying on the master branch, which seems to cause issues on composer's side (see https://github.com/composer/composer/issues/7940)

- [x] Have you updated related documentation?

#### Testing instructions (if applicable)

1. Checkout this branch, try running `make dev`.

**If you get an error about PHP and curl**, make sure you have the CURL extension enabled for your command line version of PHP.
    - To fix this on my Ubuntu VM, I ran: `sudo apt install php7.3-curl` and modified `vim /etc/php/7.3/cli/php.ini` by searching for "curl" and uncommenting the line I found.

**If you get an error from composer about 'unpushed changes'**, try:
- make clean
- composer remove facebook/webdriver
- composer update php-webdriver/webdriver

#### Link(s) to related issue(s)

* Reported in #6185 (but this does not resolve it as there is another related error)
* Resolves #6220

### CODE REVIEW

**Comment by maltheism** [comment]
Hey @johnsaigle I'm still getting the error.
```
  - Removing phar-io/manifest (1.0.1)
  - Removing phan/phan (2.4.6)
  - Removing pdepend/pdepend (2.6.1)
  - Removing ocramius/package-versions (1.5.1)
  - Removing nikic/php-parser (v4.2.5)
  - Removing nette/utils (v3.0.2)
  - Removing nette/schema (v1.0.0)
  - Removing nette/robot-loader (v3.2.0)
  - Removing nette/php-generator (v3.2.3)
  - Removing nette/neon (v3.0.0)
  - Removing nette/finder (v2.5.1)
  - Removing nette/di (v3.0.1)
  - Removing nette/bootstrap (v3.0.1)
  - Removing netresearch/jsonmapper (v1.6.0)
  - Removing myclabs/deep-copy (1.9.3)
  - Removing microsoft/tolerant-php-parser (v0.0.18)
  - Removing jean85/pretty-package-versions (1.2)
  - Removing felixfbecker/advanced-json-rpc (v3.0.4)
  - Removing facebook/webdriver (dev-master)

                                                                                                               
  [RuntimeException]                                                                                           
  Source directory /Users/alizee/Development/GitHub/McGill/Loris/vendor/facebook/webdriver has unpushed chang  
  es on the current branch:                                                                                    
  M     lib/WebDriverTargetLocator.php   
....
```

**Comment by johnsaigle** [comment]
Did you try the following steps from the PR description?

> **If you get an error from composer about 'unpushed changes'**, try:
* make clean
* composer remove facebook/webdriver
* composer update php-webdriver/webdriver

**Comment by maltheism** [comment]
Sorry @johnsaigle I didn't initially use those commands. That fixed it for me.

**Comment by driusan** [comment]
Can you send this to the 23 release?

File: composer.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "require-dev" : {
         "squizlabs/php_codesniffer" : "3.5.3",
         "phpunit/phpunit" : "7.0.0",
-        "facebook/webdriver" : "dev-master",
+        "php-webdriver/webdriver" : "^1.8.2",
```
Comment (driusan) [inline composer.json:20]: This should not be a hardcoded version number, it should be a version constraint. The `composer.lock` stores the specific version.

Comment (johnsaigle) [reply composer.json:20]: the `^` character means that it's not hard-coded though. Do you have a suggestion for an alternate version here? I don't have a preference but I'm also not sure what value would be better for you.

Comment (maltheism) [reply composer.json:20]: Maybe `@dev`? Stackoverflow answer says it will get the latest package. I'm unsure if @driusan means that though.

Comment (driusan) [reply composer.json:20]: I looked up what the '^' means.. the syntax is a little obtuse, but it looks like it's fine.

File: docs/wiki/00 - SERVER INSTALL AND CONFIGURATION/01 - LORIS Install/Ubuntu/README.md [hunk 2]
```diff
@@ -1 +1 @@
-Ubuntu specific instructions
\ No newline at end of file
+Ubuntu specific instructions
```
Comment (driusan) [inline docs/wiki/00 - SERVER INSTALL AND CONFIGURATION/01 - LORIS Install/Ubuntu/README.md:1]: why is this here/being changed?

Comment (johnsaigle) [reply docs/wiki/00 - SERVER INSTALL AND CONFIGURATION/01 - LORIS Install/Ubuntu/README.md:1]: I updated this document wiki when this was sent to `master`. On `23.0-release`, the contents are different so I reverted those changes in the rebase. I guess there was a space left over when I cleared the conflicting lines.

**Comment by kongtiaowang** [comment]
Did this PR pass Travis test?

**Comment by driusan** [comment]
@kongtiaowang I thought it had when I merged it, but I don't see the green checkmark now.. feel free to send a PR reverting it if it's breaking things
